.github/workflows/ci-changes.yml

```diff
@@ -0,0 +1,79 @@
+name: CI Changes
+on:
+  workflow_call:
```

Comment:
can it be pr triggered? want to see the results first to make sure it is working